gitstats

```diff
@@ -40,6 +40,7 @@ conf = {
     'linear_linestats': 1,
     'project_name': '',
     'exclude_extensions': 'png,gif,pdf,json,csv,jpg,sh,txt',
+    'exclude_directories': 'src/test/',
```

Comment:
This probably shouldn't have any defaults. This would be unexpected behavior for a user who didn't read the source code.